docs/6-operating-guidelines.md

```diff
@@ -24,39 +24,41 @@ Antifragility and non-gameability emerge from simple frameworks. This limits the
   - `timeWeighting` can be either 1.0 for full-time or .5 for part-time contributors
   - Existing contributors get "diluted" as newcomers come in
   - Continuing contributors get additional weight per month they are active. This means historic contributors don't maintain their split weighting if they leave protocol development
-  - Anyone who stops contributing is removed from the split via periodic curation reviews
+  - Anyone who stops contributing should remove themselves from the membership
 
-## 6.4 Proposing and Discussing New Members
+## 6.4 Modifying Projects and Members
 
-If current members believe that the set should be expanded to include another contributor, they should propose them well enough in advance of a membership update. This would mean at least a week. The process would look something like this:
+Modifying eligible projects and active membership should be proposed separately in the case both are being considered. Both should be shared with the membership at least two weeks in advance of any onchain update.
 
-1. An existing member (akin to an EIP champion) should send this info in the `proposed-members` channel.
+### Adding/removing Projects
+
+Changing the list of [eligible projects](link) can be made through a PR to the docs repo. This PR should add the project in the appropriate section, along with the following info:
+
+- Name of project/research area
+- Summary of why this project/research area should be considered eligible 
+
+### Adding Members
+
+Changing the current membership can be done by an existing member making a PR in the membership repo (currently private to members only). 
```

Comment:
We should really explore ways to make this repo public

Reply:
yep agree. - ive mentioned [it in the discord](https://discordapp.com/channels/795514951376568391/953415562153832520/1121926227876397206), but no one has taken up discussion. i will make a new issue discussing the pros and cons